design/TermThresholdDesign.md

```diff
@@ -0,0 +1,108 @@
+# Term Threshold
+Presently, recommendations are categorized into Short, Medium, and Long terms based on the duration of historical data considered.
```

Comment:
You can add the issue due to this and mention about enforcing the term duration here